### PR TITLE
Added captains and updated prize for CWC

### PR DESCRIPTION
**Pull request by juankristal**

### CODE REVIEW

File: wiki/Tournaments/CWC/2018/en.md [hunk 1]
```diff
@@ -52,13 +50,45 @@ The osu!catch World Cup 2018 is run by various community members by distributing
...
+| ![][flag_MY] | **Malaysia** | **[Kagari](https://osu.ppy.sh/u/3366264)** |
+| ![][flag_NL] | **Netherlands** | **[SartNYAN OwO xD](https://osu.ppy.sh/u/4100941)** |
+| ![][flag_NO] | **Norway** | **[Cameragod](https://osu.ppy.sh/u/4974088)** |
+| ![][flag_PH] | **Philippines** | **[-Plus-](https://osu.ppy.sh/u/829284)** |
+| ![][flag_PL] | **Poland** | **[Kosmit](https://osu.ppy.sh/u/1749173)** |
+| ![][flag_PT] | **Portugal** | **[Buffow](https://osu.ppy.sh/u/4427038)** |
+| ![][flag_RU] | **Russian Federation** | **[Myle](https://osu.ppy.sh/u/4915649)** |
+| ![][flag_SE] | **Sweden** | **[Setomi](https://osu.ppy.sh/u/2205929)** |
+| ![][flag_SG] | **Singapore** | **[He Ang Erika](https://osu.ppy.sh/u/2451381)** |
+| ![][flag_TH] | **Venezuela** | **[-Luminate](https://osu.ppy.sh/u/4778689)** |
```
Comment (WalterToro) [inline wiki/Tournaments/CWC/2018/en.md]: Should be Taiwan, not Venezuela.

File: wiki/Tournaments/CWC/2018/en.md [hunk 2]
```diff
@@ -52,13 +50,45 @@ The osu!catch World Cup 2018 is run by various community members by distributing
...
-- **[Sign up here!](https://osu.ppy.sh/community/tournaments/15)**
 
 ------------------------------------------------------------------------
 
 ## Participants
 
-_The participant list will be revealed once all sign ups are processed, this could take at most one week after the sign ups phase is concluded_
+| | Country | Members |
+| ---: | :---: | :--- |
+| ![][flag_AR] | **Argentina** | **[[Force]](https://osu.ppy.sh/u/1617217)** |
```
Comment (WalterToro) [inline wiki/Tournaments/CWC/2018/en.md]: All user links should use /users/ instead of /u/

File: wiki/Tournaments/CWC/2018/en.md [hunk 3]
```diff
@@ -52,13 +50,45 @@ The osu!catch World Cup 2018 is run by various community members by distributing
...
+| ![][flag_PH] | **Philippines** | **[-Plus-](https://osu.ppy.sh/u/829284)** |
+| ![][flag_PL] | **Poland** | **[Kosmit](https://osu.ppy.sh/u/1749173)** |
+| ![][flag_PT] | **Portugal** | **[Buffow](https://osu.ppy.sh/u/4427038)** |
+| ![][flag_RU] | **Russian Federation** | **[Myle](https://osu.ppy.sh/u/4915649)** |
+| ![][flag_SE] | **Sweden** | **[Setomi](https://osu.ppy.sh/u/2205929)** |
+| ![][flag_SG] | **Singapore** | **[He Ang Erika](https://osu.ppy.sh/u/2451381)** |
+| ![][flag_TH] | **Venezuela** | **[-Luminate](https://osu.ppy.sh/u/4778689)** |
+| ![][flag_TW] | **Taiwan** | **[XXXZ](https://osu.ppy.sh/u/2111624)** |
+| ![][flag_US] | **United States** | **[Zak](https://osu.ppy.sh/u/1375955)** |
+| ![][flag_VN] | **Venezuela** | **[Hana Yukiko](https://osu.ppy.sh/u/4964095)** |
```
Comment (WalterToro) [inline wiki/Tournaments/CWC/2018/en.md]: Should be Vietnam, not Venezuela

File: wiki/Tournaments/CWC/2018/en.md (hunk 1 above)
Comment (juankristal) [reply wiki/Tournaments/CWC/2018/en.md]: thailand, but yes

**Pull request merged**